index.html

```diff
@@ -36,7 +36,186 @@ <h2>
+                    <ul>
+                    <h3>html</h3>
+                    <h3>bootstrap</h3>
+                    <h3>Ruby</h3>
+                </ul>
+                </div>
+                
+            </div>
+            <button>See Project</button>
+            <div class="cards">
+                <h2>
+                    Profesional Art Printing Data
+                </h2>
+                <p>
+                    A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required. has been the industry's standard
+                </p>
+                <div>
+                <ul>
+                    <h3>html</h3>
+                    <h3>bootstrap</h3>
+                    <h3>Ruby</h3>
+                </ul>
+                </div>
+                
+            </div>
+            <button>See Project</button>
+            <div class="cards">
+                <h2>
+                    Profesional Art Printing Data
+                </h2>
+                <p>
+                    A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required. has been the industry's standard
+                </p>
+                <div>
+                    <ul>
+                    <h3>html</h3>
+                    <h3>bootstrap</h3>
+                    <h3>Ruby</h3>
+                </ul>
+                </div>
+                
+            </div>
+            <button>See Project</button>
+            <div class="cards">
+                <h2>
+                    Profesional Art Printing Data
+                </h2>
+                <p>
+                    A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required. has been the industry's standard
+                </p>
+                <div>
+                    <ul>
+                    <h3>html</h3>
+                    <h3>bootstrap</h3>
+                    <h3>Ruby</h3>
+                </ul>
+                </div>
+                
+            </div>
+            <button>See Project</button>
+        </section>
+        <section class="About">
+            <div>
+                <h2>
+                    About me
+                </h2>
+                <p>
+                    Hello I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <button>
+                    Get My Resume
+                </button>
+            </div>
+            <div class="properties">
+                <ul class="lang">
+                <div>
+                    <ul>
+                    <h3>Language</h3>
+                    <h4>Javascript</h4>
+                    <h4>Ruby</h4>
+                    <h4>Html</h4>
+                    <h4>Css</h4>
```

Comment:
- [ ] As requested by the previous reviewer please use a `ul` and replace every `h4` with `li`, the same goes for the Frameworks list and Skills list.